_includes/video/pb-is-bc.html

```diff
@@ -3,8 +3,7 @@
 
 <head>
 {% include head--common.html %}
-    <!--production version of prebid.js-->
-    <script async src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js"></script>
+    {% include prebidjs-non-prod.html %}
 
     <!--flowplayer & prebid js code-->
     <script type="text/plain" class="optanon-category-C0003">
```

Comment:
Isn't optanon the onetrust code? Missing the new approach here?

Reply:
It is! And one can make this a simple script tag again, because there are no external calls